ridemyway/tests/integration/vehicle/test_vehicle.py

```diff
@@ -0,0 +1,47 @@
+from ridemyway.tests.base_test_case import VehicleBaseTestCase
+
+
+class TestVehicle(VehicleBaseTestCase):
+    def test_create_vehicle(self):
```

Comment:
I think we can add on the test names whether it succeeds or fails eg `test_create_vehicle_succeeds`

Reply:
Will add test cases with invalid or empty fields.